README.md

```diff
@@ -22,9 +27,76 @@ Code quality checking and enforcement is done with the following tools:
 
 Error and exception logging is done using Sentry.
 
-## Deployment
- - Provide `SENTRY_DSN` and `SENTRY_ENVIRONMENT` as environmental variables
-   during deployment.
+#### Environmental Variables
+
+| Name                                  | Description                                  | Default   |
+|---------------------------------------|----------------------------------------------|-----------|
+| DB_HOST                               | The MongoDB host to connect to.              | localhost |
+| DB_PORT                               | The port to connect to MongoDB on.           | 27017     |
+| DB_USER                               | The username to access the MongoDB instance. | admin     |
+| DB_NAME                               | The name of the MongoDB instance.            | forms     |
+| DB_PASSWORD                           | The password to access the MongoDB instance. | pwd       |
+| SENTRY_DSN                            | A Sentry error monitoring Data Source Name.  |           |
+| SENTRY_ENVIRONMENT                    | A Sentry error monitoring DSN environment.   | local     |
+| FEATURE_FORMR_PARTB_COVID_DECLARATION |                                              | true      |
+| APPLICATION_FILESTORE_BUCKET          | the S3 bucket the forms are stored in.       |           |
+| APPLICATION_FILESTORE_ALWAYSSTORE     |                                              | false     |
+
+## Usage
+### Saving Forms
+
+To save the form that is received a key is made from the ObjectKeyTemplate, TraineeTisID and form
```

Comment:
The ObjectKeyTemplate is just the format in which to write the TraineeTisId and form Id, so for FormR PartA it's "`%s/forms/formr-a/%s.json`"; so if the TraineeTisId = 1 and the form Id = 2 it would give "`1/forms/formr-a/2.json`";